pandas/tests/dtypes/test_dtypes.py

```diff
@@ -323,6 +324,13 @@ def test_tz_standardize(self):
         dtype = DatetimeTZDtype("ns", dr[0].tz)
         assert dtype.tz == tz
 
+    def test_dfgroupby_shift_tz(self):
+        # GH 30134
+        df = pd.DataFrame({"a": [1, 1], "date": dt.datetime.now(tz=pytz.utc)})
+        expected = df.dtypes["date"]
+        result = df.groupby("a").shift(0).dtypes["date"]
+        assert expected == result
```

Comment:
Can you build the actual expected series and use `tm.assert_series_equal(result, expected)`? Should see examples of this throughout tests